assets/src/edit-story/components/sidebar/provider.js

```diff
@@ -47,25 +46,20 @@ const SidebarContent = styled.div`
   top: ${({ top }) => `${top}px`};
 `;
 
-const TYPE_COLORPICKER = 'colorpicker';
-
 function SidebarProvider({ children }) {
   const [sidebarState, setSidebarState] = useState(null);
 
   const hasSidebar = sidebarState !== null;
   const offset = hasSidebar && sidebarState.offset;
-  const type = hasSidebar && sidebarState.type;
-  const props = hasSidebar && sidebarState.props;
 
   const ref = useRef();
 
-  const showColorPickerAt = useCallback((node, colorProps) => {
-    const colorOffset =
+  const showSidebarAt = useCallback((node, displayContent) => {
```

Comment:
Rename it to `renderContent` - that'll make it more transparent as a render prop style structure.